packages/venia-ui/lib/components/CartPage/ProductListing/product.js

```diff
@@ -102,6 +109,33 @@ const Product = props => {
               })
             : '';
 
+    let multipleWishlistDialog = null;
+    if (isMultipleWishlistsEnabled) {
+        const sku = item.product.sku;
+        const quantity = item.quantity;
+        const selected_options = item.configurable_options
+            ? item.configurable_options.map(
+                  option => option.configurable_product_option_value_uid
+              )
+            : [];
+
+        multipleWishlistDialog = (
+            <WishlistDialog
+                isOpen={isWishlistDialogOpen}
+                itemOptions={{
+                    sku,
+                    quantity,
+                    selected_options
+                }}
+                onClose={handleWishlistDialogClose}
+                onSuccess={handleAddToWishlistSuccess}
+                isLoading={isProductUpdating}
+            />
+        );
+    } else {
```

Comment:
you don't need you to init multipleWishlistDialog with `let multipleWishlistDialog = null;`